content/doc/addons/postgresql/_index.md

```diff
@@ -18,42 +18,38 @@ aliases:
 type: docs
 ---
 
-PostgreSQL is an object-relational database management system (ORDBMS) with an emphasis on extensibility
-and on standards-compliance.
+PostgreSQL is an object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance.
 
 ## Versions
 
-The version currently installed by the add-on is :
-
-- on shared plans (DEV) : PostgreSQL 15
-- on newly created dedicated databases (plans XS Small Space and above) : PostgreSQL 10, 11, 12, 13, 14, 15
+Versions currently available for order are as follows:
+- On shared (DEV) cluster: PostgreSQL version 15
+- On dedicated plans: PostgreSQL versions 10, 11, 12, 13, 14 and 15
 
 {{< readfile file="db-backup.md" >}}
 
 ## Migrating from an old database
 
-Some applications require a populated database to run properly.
-If you want to import your **SQL** dump, you can use several methods:
-
-1. [Our WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/).
-2. Command line tool for PostgreSQL administration like `psql`.
-3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/).
+Some applications require a non-empty database to run properly. If you want to import your **SQL** dump, you can use several methods:
+1. [WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/)
+2. Command line tool for PostgreSQL administration like `psql`
+3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/)
 
 ## Direct access
 
-All our dedicated PostgreSQL databases are served via a proxy. To reduce the latency you can bypass this proxy by generating direct hostname and port for the addon. You can do it by clicking the "Generate direct hostname and port" on the addon dashboard.
-
-This action will add new environment variables to reach the addon without any proxy.
+A proxy serves all dedicated PostgreSQL databases. In some cases, this can add some latency between applications and their database. If this is an issue, you must generate a direct hostname and port for the add-on to bypass the proxy, using the "Generate direct hostname and port" button in the add-on dashboard.
```

Comment:
```suggestion
A proxy serves all dedicated PostgreSQL databases. In some cases, this can add some latency between applications and their database. If this is an issue, you can generate a direct hostname and port for the add-on to bypass the proxy, using the "Generate direct hostname and port" button in the add-on dashboard.
```